src/overlays/actors/ovl_En_Dnk/z_en_dnk.h

```diff
@@ -24,9 +27,19 @@ typedef struct EnDnk {
     /* 0x188 */ EnDnkActionFunc actionFunc;
     /* 0x18C */ UNK_TYPE1 unk_18C[0x4];
     /* 0x190 */ ColliderCylinder collider;
-    /* 0x1DC */ Vec3s jointTable[11];
-    /* 0x21E */ Vec3s morphTable[11];
-    /* 0x260 */ Gfx* unk_260[11];
+    /* 0x1DC */ union {
+                    Vec3s jointTable[1]; // Union used to ensure size is always max of the three LIMB_MAX
+                    Vec3s jointTableGuard[DEKU_PALACE_GUARD_LIMB_MAX];
+                    Vec3s jointTableHint[OBJECT_HINTNUTS_LIMB_MAX];
+                    Vec3s jointTableScrub[DEKU_SCRUB_LIMB_MAX];
+                };
+    /* 0x21E */ union {
+                    Vec3s morphTable[1]; // Union used to ensure size is always max of the three LIMB_MAX
+                    Vec3s morphTableGuard[DEKU_PALACE_GUARD_LIMB_MAX];
+                    Vec3s morphTableHint[OBJECT_HINTNUTS_LIMB_MAX];
+                    Vec3s morphTableScrub[DEKU_SCRUB_LIMB_MAX];
+                };
```

Comment:
Does this seem like a reasonable approach when the jointTable is shared among three different skeletons depending on the type?

Reply:
I've generally just created a define for the actor that takes the max of the limbs

`#define ENSOB1_LIMB_MAX MAX(MAX((s32)ZORA_LIMB_MAX, (s32)BOMB_SHOPKEEPER_LIMB_MAX), (s32)GORON_LIMB_MAX)`
Note the `s32` casts are to suppress a warning.

Reply:
Nice, I like that better

Reply:
Ohhh, this is an interesting approach, but the `MAX` seems better tho. This approach may be useful in the future